docker-compose.yml.example

```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
```

Comment:
Can you elaborate? This is an example `docker-compose.yml` since this file is only used for local testing and this compose file is being used to start any provider and consumer, which would include SSC/SPC as well.